plotting/comparison.py

```diff
@@ -55,6 +60,15 @@
     type=str,
     help="variables to plot, splitted by ,. Wildcard option * available as well. Specifying `all` will run through all variables.",
 )
+parser.add_argument(
+    "--xrange", type=str, default=None, help="custom x-range, --xrange xmin,xmax"
```

Comment:
general comment: add ylabel argument and usage of xerr bars in case of variable bin width as done in plotdatMC.py